tests/drivers/i2s/i2s_api/src/i2s_api_test.h

```diff
@@ -1,12 +1,13 @@
 /*
  * Copyright (c) 2017 comsuisse AG
  * Copyright (c) 2018 Intel Corporation
+ * Copyright (c) 2021 NXP
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef I2S_API_TEST_H
-#define I2S_API_TEST_H
+#ifndef _I2S_API_TEST_H_
+#define _I2S_API_TEST_H_
```

Comment:
Why is this change made

Reply:
just a coding style alignment, and I find there is no leading '_' now, let me change to I2S_API_TEST_H_